netty/src/main/java/io/grpc/netty/GrpcSslContexts.java

```diff
@@ -196,7 +197,14 @@ public static SslContextBuilder configure(SslContextBuilder builder, Provider jd
         apc = ALPN;
       } else {
         throw new IllegalArgumentException(
-            SUN_PROVIDER_NAME + " selected, but Java 9+ and Jetty NPN/ALPN unavailable");
+            jdkProvider.getName() + " selected, but Java 9+ and Jetty NPN/ALPN unavailable");
+      }
+    } else if (IBM_PROVIDER_NAME.equals(jdkProvider.getName())) {
+      if (JettyTlsUtil.isJava9AlpnAvailable()) {
+        apc = ALPN;
+      } else {
+        throw new IllegalArgumentException(
```

Comment:
Similar to my other comment. Because of where this else clause is placed the exception thrown will be different when the `jdkProvider` is IBMJSSE2 and Java9Alpn is unavailable. Most probably it's not a big deal but I thought it's worth pointing out.

Reply:
It should be different, as Jetty ALPN is not supported with IBMJSSE2.